src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/stores/DatagridStore.js

```diff
@@ -44,6 +44,10 @@ export default class DatagridStore {
         this.setPage(1);
         this.loadingStrategy = loadingStrategy;
 
+        if (this.localeInterceptionDisposer) {
+            this.localeInterceptionDisposer();
+        }
+
         if ('infiniteScroll' === this.loadingStrategy && !this.localeInterceptionDisposer) {
```

Comment:
then the check `!this.localeInterceptionDisposer` isn't needed anymore

Reply:
True, just removed it.